mmil.raw.html

```diff
@@ -11455,6 +11461,49 @@
   <td>the set.mm proof uses sineq0</td>
 </tr>
 
+<tr>
+  <td>cosord</td>
+  <td>~ cosordlem , ~ cos11</td>
+  <td>presumably provable but the set.mm proof
+  uses real number trichotomy</td>
+</tr>
+
+<tr>
+  <td>sinord</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof depends on cosord</td>
+</tr>
+
+<tr>
+  <td>recosf1o</td>
+  <td>~ ioocosf1o</td>
+  <td>the set.mm proof of recosf1o relies on ivthle2</td>
+</tr>
```

Comment:
Isn't there any other way to go around this?

Since you have the strict version of the intermediate value theorem, can't you e.g. at least prove a similar theorem on open intervals?

Reply:
ioocosf1o IS a similar theorem on open intervals.

I suspect that might not be enough long-term. But it is what I could clearly see how to prove, and included in this pull request.